src/pegin/components/create/ConfirmTx.vue

```diff
@@ -146,13 +141,13 @@ export default defineComponent({
       senderAddress: pegInTxState.value.normalizedTx.inputs[0].address,
     }));
 
-    const flyoverTotalFee = computed(() => getProviderFee()
+    const flyoverTotalFee = computed(() => selectedQuote.value.providerFee
       .plus(safeFee.value));
 
     const flyoverPeginSummary = computed((): NormalizedSummary => ({
       amountFromString: selectedQuote.value.quote.value.toBTCTrimmedString(),
       amountReceivedString: selectedQuote.value.quote.value.toBTCTrimmedString(),
-      fee: Number(flyoverTotalFee.value),
+      fee: Number(flyoverTotalFee.value.toBTCTrimmedString()),
```

Comment:
flyoverTotalFee needs to be on selectedQuote object.

Reply:
I'm assuming the quote is not responsable of the networkFee calculation, here we called `safeFee` and its outside of the Quote